gensim/models/ensemblelda.py

```diff
@@ -0,0 +1,1315 @@
+
+For more details read our paper (https://www.hip70890b.de/machine_learning/ensemble_LDA/).
+
+Usage examples
+--------------
+
+Train an ensemble of LdaModels using a Gensim corpus
+
+.. sourcecode:: pycon
+
+    >>> from gensim.test.utils import common_texts
+    >>> from gensim.corpora.dictionary import Dictionary
+    >>> from gensim.models import EnsembleLda
+    >>>
+    >>> # Create a corpus from a list of texts
+    >>> common_dictionary = Dictionary(common_texts)
+    >>> common_corpus = [common_dictionary.doc2bow(text) for text in common_texts]
+    >>>
+    >>> # Train the model on the corpus. corpus has to be provided as a
+    >>> # keyword argument, as they are passed through to the children.
+    >>> elda = EnsembleLda(corpus=common_corpus, id2word=common_dictionary, num_topics=10, num_models=4)
+
+Save a model to disk, or reload a pre-trained model
+
+.. sourcecode:: pycon
+
+    >>> from gensim.test.utils import datapath
+    >>>
+    >>> # Save model to disk.
+    >>> temp_file = datapath("model")
+    >>> elda.save(temp_file)
+    >>>
+    >>> # Load a potentially pretrained model from disk.
+    >>> elda = EnsembleLda.load(temp_file)
+
+Query, the model using new, unseen documents
+
+.. sourcecode:: pycon
+
+    >>> # Create a new corpus, made of previously unseen documents.
+    >>> other_texts = [
+    ...     ['computer', 'time', 'graph'],
+    ...     ['survey', 'response', 'eps'],
+    ...     ['human', 'system', 'computer']
+    ... ]
+    >>> other_corpus = [common_dictionary.doc2bow(text) for text in other_texts]
+    >>>
+    >>> unseen_doc = other_corpus[0]
+    >>> vector = elda[unseen_doc]  # get topic probability distribution for a document
+
+Increase the ensemble size by adding a new model. Make sure it uses the same dictionary
+
+.. sourcecode:: pycon
+
+    >>> from gensim.models import LdaModel
+    >>> elda.add_model(LdaModel(common_corpus, id2word=common_dictionary, num_topics=10))
+    >>> elda.recluster()
+    >>> vector = elda[unseen_doc]
+
+To optimize the ensemble for your specific case, the children can be clustered again using
+different hyperparameters
+
+.. sourcecode:: pycon
+
+    >>> elda.recluster(eps=0.2)
+
+References
+----------
+.. [1] REHUREK, Radim and Sojka, PETR, 2010, Software framework for topic modelling with large corpora. In : THE LREC
+       2010 WORKSHOP ON NEW CHALLENGES FOR NLP FRAMEWORKS [online]. Msida : University of Malta. 2010. p. 45-50.
+       Available from: http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.695.4595
+
+.. [2] BRIGL, Tobias, 2019, Extracting Reliable Topics using Ensemble Latent Dirichlet Allocation [Bachelor Thesis].
+       Technische Hochschule Ingolstadt. Munich: Data Reply GmbH. Available from:
+       https://www.hip70890b.de/machine_learning/ensemble_LDA/
+
+Citation
+--------
+At the moment, there is no paper associated to ensemble LDA but we are working on publicly releasing Tobi Brigl's
```

Comment:
Above you say `For more details read our paper `.

Reply:
yeah, the idea was to publish it when the PR is merged to make sure the information in there is really matching the code. I'll do that timely after the merge or maybe even in a few days, I think we should already put the correct text there.